bpf-metrics-exporter/README.md

```diff
@@ -14,24 +14,32 @@ to correlate process IDs -> containers -> k8s pods.
 
 The following metrics are exported:
 
-- `bpf_program_size_jitted_bytes`: The size of the BPF program in bytes.
-- `bpf_program_size_translated_bytes`: The size of the BPF program in bytes.
-- `bpf_program_mem_bytes`: The amount of memory used by the BPF program in bytes.
-- `bpf_program_verified_instructions_total`: The number of instructions in the BPF program.
-
-
-The following will be added pending: https://github.com/open-telemetry/opentelemetry-rust/issues/1242
-
-- `bpf_programs_total`: The number of BPF programs loaded into the kernel.
-
-Labels:
-
-- `id`: The ID of the BPF program
-- `name`: The name of the BPF program
-- `type`: The type of the BPF program
-- `tag`: The tag of the BPF program
-- `gpl_compatible`: Whether the BPF program is GPL compatible
-- `load_time`: The time the BPF program was loaded
+- `bpf_programs`: Information on each loaded BPF Program
```

Comment:
If we're doing this, I'd rename this metric to `bpf_program_{N}_info`, where `N` is the program ID and the value is always 1. See: https://www.robustperception.io/exposing-the-software-version-to-prometheus/